### PR TITLE
updated documentation

### PR DESCRIPTION
**Pull request by John-Ragland**
from Issue #70 

Made a couple of changes to documentation and removed out of date notebooks. Let me know what you think.

### CODE REVIEW

File: hydrophone_demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download Data from OOI\n",
+    "\n",
+    "### Valid Hydrophone Nodes: \n",
+    "#### Broadband\n",
+    "- '\\LJ01D' - Oregon Shelf Base Seafloor\n",
```
Comment (FelixSchwock) [inline hydrophone_demo.ipynb:54]: I think we should remove the "\" here (and in the next few lines) to not confuse people

**Review by FelixSchwock — CHANGES_REQUESTED** [review]

File: hydrophone_demo.ipynb (hunk 1 above)
Comment (FelixSchwock) [reply hydrophone_demo.ipynb:54]: Also it seems that the PEP8 failed due to an install error. It seems to use python 3.7.10 but PEP8 only supports 3.7.11 or some other versions. Do you have any idea how that happened?

Comment (FelixSchwock) [reply hydrophone_demo.ipynb:54]: ok I think I soled the python version issue in the other PR

Comment (John-Ragland) [reply hydrophone_demo.ipynb]: I updated the demo with the same info from readthedocs, so it should be clear now.

**Review by FelixSchwock — APPROVED** [review]

**Pull request merged**